userguide/docs/faq.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 4
+---
+
+# Frequently Asked Questions
+
+This section aims to answer some common questions as well as covering some pitfalls you might experience when using the NeoForge installer.
+
+## Troubleshooting the NeoForge Installer
+
+For most users the installer should run without error.
+
+If the installer runs but pops up an error dialog box while executing, see the [NeoForge Installer Log File] section below.
```

Comment:
This introduction can be omitted imo. Maybe mention the "If the installer gives you an error box" part in the NeoForge Log Files section below.